mackup/applications/istat-menus.cfg

```diff
@@ -0,0 +1,9 @@
+[application]
+name = iStat Menus (5 and 6)
```

Comment:
Can you not specify the version to keep it clean?
Not sure why it was specified in the file name in the 1st place.

Thanks!